cluster/mesos/docker/util.sh

```diff
@@ -256,7 +254,7 @@ function cluster::mesos::docker::init_auth {
 function kube-up {
   # Nuke old mesos-slave workspaces
   for ((i=1; i <= NUM_MINIONS; i++)) do
```

Comment:
likely want to nuke all the dirs, not just the ones we're currently configured for. ex: downsizing